### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by HySpeed**
updated from feedback

### CODE REVIEW

File: scripts/coe_player.lua [hunk 1]
```diff
@@ -113,7 +113,6 @@ function Player.onPlayerCreated(event)
   local player = game.get_player(event.player_index)
   if Utils.getStartupSetting("coe_dev_mode") then setupForDevMode(player) end
   Utils.print("Player " .. player.name .. " created.", true)
-
```
Comment (Nexela) [inline scripts/coe_player.lua:116]: Blank line still snuck in.

Comment (HySpeed) [reply scripts/coe_player.lua:116]: oh, I thought you wanted the blank line. :(

**Comment by Nexela** [comment]
I might have said I wanted the wrong thing

File: scripts/coe_player.lua (hunk 1 above)
Comment (HySpeed) [reply scripts/coe_player.lua:116]: yeah, I'm confused - my code does not have a blank line after the print.  Do you want one there?

File: .vscode/settings.json [hunk 2]
```diff
@@ -77,5 +77,10 @@
     "data/world_americas.lua",
     "old/",
     "base/scenarios/"
+  ],
+  "Lua.diagnostics.globals": [
```
[the remark this answers is not in the document]
Comment (HySpeed) [reply .vscode/settings.json:81]: I'll just get the one from the main branch.

File: scripts/coe_player.lua (hunk 1 above)
Comment (Nexela) [reply scripts/coe_player.lua:116]: Meh its not hurting anything I guess it can stay

**Pull request merged**